packages/components/src/components/buttons/Button/Button.tsx

```diff
@@ -16,7 +16,7 @@ import { makePropsTransient, TransientProps } from '../../../utils/transientProp
 import { FrameProps, FramePropsKeys, withFrameProps } from '../../../utils/frameProps';
 import { Icon, IconName } from '../../Icon/Icon';
 
-export const allowedButtonFrameProps: FramePropsKeys[] = ['margin'];
+export const allowedButtonFrameProps = ['margin', 'minWidth'] as const satisfies FramePropsKeys[];
```

Comment:
This needs to be inferred type, not declarative `FramePropsKeys[]`.
Otherwise, `(typeof allowedButtonFrameProps)[number] = FramePropsKeys` 
and  `Pick<FrameProps, (typeof allowedButtonFrameProps)[number]> = FrameProps`

But we can use `satisfies FramePropsKeys[]` to type check that it doesn't contain nonsense.